__init__.py

```diff
@@ -831,9 +853,8 @@ def draw(self, context):
         row.prop(context.scene, 'line_in_debug_print')
         row.operator(DEV_OT_updateDebugLinum.bl_idname, text='', icon='FILE_REFRESH')#Update
         layout.operator(DEV_OT_debugPrintVariable.bl_idname)
-        layout.separator()
-        layout.operator(DEV_OT_disableAllDebugPrint.bl_idname)
-        layout.operator(DEV_OT_enableAllDebugPrint.bl_idname)
+        layout.label(text="Options")
+        layout.prop(context.scene, 'enum_DebugPrint', text='', expand=False)
```

Comment:
I like what you have done with this idea of compacting the UI a bit.
But the thing that's bothering me is that the action is less clear for the user.  The unconventional use of enum with the _props update_ hack make it display the last action and doesn't implicitly show that it's an instant choice.
(and another personal preferences, I'm not big fan of using eval in general, but's that's not relevant here)

I'll accept the request since I'm happy to see contributors out there ^^. But I wanted to share my thoughts with you and let you know that I might tinker with it a bit later (too busy these days, that's why I delayed my check).

First thing that come to my mind is "completing" your hack by adding a 4rth entry to the enum that serve only as descriptive label (like "_debug print actions_"), but at each use, the enum display should be reset to this value (maybe possible in the update... don't know yet).

Thank you for your contribution.